tests/functional/base_test.go

```diff
@@ -160,88 +160,6 @@ func GetOVNDBCluster(name types.NamespacedName) *ovnv1.OVNDBCluster {
 	return instance
 }
 
-func SimulateStatefulSetReplicaReadyWithPods(name types.NamespacedName, networkIPs map[string][]string) {
-	ss := th.GetStatefulSet(name)
-
-	for i := 0; i < int(*ss.Spec.Replicas); i++ {
-		pod := &corev1.Pod{
-			ObjectMeta: ss.Spec.Template.ObjectMeta,
-			Spec:       ss.Spec.Template.Spec,
-		}
-		pod.ObjectMeta.Namespace = name.Namespace
-		pod.ObjectMeta.GenerateName = name.Name
-		// Hack to avoid getting dynamic volume mount for etc-ovn
-		pod.Spec.Volumes = []corev1.Volume{}
-		pod.Spec.Containers[0].VolumeMounts = []corev1.VolumeMount{}
```

Comment:
I think this is a generic solution for operators using PVCs, so I moved this to lib-common